edgePy/DGEList.py

```diff
@@ -357,6 +357,48 @@ def rpkm(
 
         return self.copy(counts=counts, current_log=current_log, genes=genes)
 
+    def tpm(
+        self,
+        gene_data: CanonicalDataStore,
+        transform_to_log: bool = False,
+        prior_count: float = PRIOR_COUNT,
+    ) -> "DGEList":
+        """Return the DGEList normalized to transcripts per million reads.
+        TPM =   countsPerBase * ( 1/sum[countsPerBase]) * 10^6
```

Comment:
In my commented tpm function you'll see how to do this with latex/sphinx so it's nicer in the docs.